src/ray/stats/metric_defs.cc

```diff
@@ -124,6 +124,11 @@ DEFINE_stats(scheduler_placement_time_s,
              ("WorkloadType"),
              ({0.1, 1, 10, 100, 1000, 10000}, ),
              ray::stats::HISTOGRAM);
+DEFINE_stats(local_resource_view_node_count,
+             "Number of nodes tracked in the ClusterResourceManager.",
+             (),
+             (),
+             ray::stats::GAUGE);
```

Comment:
feels a little too tightly coupled to implementation details

maybe something like:
```suggestion
DEFINE_stats(num_known_nodes_in_cluster,
             "Number of nodes tracked in the ClusterResourceManager.",
             (),
             (),
             ray::stats::GAUGE);
```

Reply:
also -- I think by prometheus best practices, this metric should not include "count" in the name because it's a gauge (which means it does not monotonically increase)